src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1823,7 +1823,7 @@ public void run() {
                         }
 
                         final InAppNotification.Type inAppType = toShow.getType();
-                        if (inAppType == InAppNotification.Type.TAKEOVER) {
+                        if (inAppType == InAppNotification.Type.TAKEOVER  && ! ConfigurationChecker.checkTakeoverInAppActivityAvailable(parent.getApplicationContext())) {
```

Comment:
Should we cache checker result? Operations with `PackageManager` are not lightweight and it's nonsense that registered manifest activities will be changed at run-time.